app/models/direct_file_data.rb

```diff
@@ -616,10 +616,27 @@ def dependents
         unless self.mailing_state == 'AZ'
           dependent.months_in_home = eitc_dependent_node.at('MonthsChildLivedWithYouCnt')&.text.to_i
         end
-        dependent.eic_student = eitc_dependent_node.at('ChildIsAStudentUnder24Ind')&.text == "true"
-        dependent.eic_disability = eitc_dependent_node.at('ChildPermanentlyDisabledInd')&.text == "true"
+        dependent.eic_student = case eitc_dependent_node.at('ChildIsAStudentUnder24Ind')&.text
+                                when 'true'
```

Comment:
nice! Wonder if we can break this out to a separate method. But also fine to keep it here

Reply:
Good idea, I'll break it up!